en/components/grid_column_resizing.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Grid column deferred resizing
+_description: The Ignite UI for Angular Data Grid control features the fastest, touch-responsive data-rich grid with popular features, including hierarchical and list views.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Data Grid component, Angular Data Grid control, Angular Grid component, Angular Grid control, Angular High Performance Grid, column resizing, deferred column reszing
+---
+
+### Grid Column Resizing
+
+**Column resizing** is also enabled per-column level, meaning that the **igx-grid** can have a mix of resizable and non-resizable columns. This is done via the `resizable` input of the `igx-column`. The **igx-grid** supports deferred column resizing, meaning that columns are not immediately resized. A temporary resize indicator is shown while the drag operation is in effect and the new column size is only applied once the drag operation ends. Resizing a certain column affects the overall width of the grid meaning that the rest of the columns will preserve their current widths.
```

Comment:
"meaning that" is used two times, the second time may be replaced with so or something similar.